jobs/label_misinformation/app/s3_utils.py

```diff
@@ -144,12 +147,14 @@ def reformat_and_save(df, output_folder="output_json_files") -> str:
 
 # one json file per json row
 def save_json(
-    df: pd.DataFrame, channel: str, date: pd.Timestamp, s3_path, folder_inside_bucket=None
+    df: pd.DataFrame, channel: str, date: pd.Timestamp, s3_path, folder_inside_bucket=None, country:Union[Country, CountryCollection]=FRANCE_COUNTRY
 ) -> str:
     based_path = "s3"
 
     local_json = "s3/json"
     if folder_inside_bucket is not None:
+        if country not in LEGACY_COUNTRIES:
+            folder_inside_bucket = f"country={country.name}/" + folder_inside_bucket
```

Comment:
would it make sense to use this function here ?

```suggestion
            folder_inside_bucket =get_bucket_key_folder(date, channel, root_folder=folder_inside_bucket, country=country)
```